common.sh

```diff
@@ -19,34 +19,34 @@ function setup_docker_machine() {
 
         # Check if the docker machine exists already, create one if not.
         if [[ $? == 1 ]]; then
-            notice "No ${MACHINE} environment found"
+            echo "No ${MACHINE} environment found"
```

Comment:
I'm guessing the method notice is missing the common.sh. notice is lot more friendlier than echo. I suggest that we pull in the code that makes it work. @singularo @pingers